BookingsApi/BookingsApi.UnitTests/Validation/V1/RepresentativeValidationTests .cs

```diff
@@ -32,18 +32,15 @@ public async Task Should_return_missing_representee_error()
         }
      
         [Test]
-        public async Task Should_return_missing_organisation_error()
+        public async Task Should_not_return_missing_organisation_error()
         {
```

Comment:
What's the difference between this test and the one above it in terms of setup? can't see why one be valid and one isnt?